tests/unit/Makefile.am

```diff
@@ -12,7 +12,8 @@ tests_unit_TESTS			= \
 	tests/unit/test_zone		   \
 	tests/unit/test_pathutils	   \
 	tests/unit/test_logwriter	\
-	tests/unit/test_thread_wakeup
+	tests/unit/test_thread_wakeup	\
+	tests/unit/test_evt_tag_mem
```

Comment:
Thank you for the tests.

There is ongoing work about moving all the remaining unit tests from `tests/unit` to near to the tested unit (#4046).

For example, `lib/eventlog/tests` might be a better place.

(eventlog used to be a separate library maintained in a separate repository, but we've merged it into the syslog-ng repo after finding out that eventlog did not become too widespread. For that reason, you won't find Criterion-based tests on that directory, but I think it is absolutely okay if we add one.)

Reply:
Thanks for the clarification, I will move it to `lib/eventlog/tests/`

Reply:
Thanks. There are currently 2 differently licensed tests with the same content under `lib/eventlog/tests/`. Please check it, it's probably just a rebase issue.